napari/_vispy/canvas.py

```diff
@@ -436,6 +436,41 @@ def _on_mouse_move(self, event: MouseEvent) -> None:
         -------
         None
         """
+        layer = self.viewer.layers.selection.active
+        visual_layer = self.layer_to_visual[layer]
+        if hasattr(visual_layer, "_face_picking_filter"):
+            render_size = tuple(
+                d * self._scene_canvas.pixel_scale
+                for d in self._scene_canvas.size
+            )
+            x_pos = event.pos[0] * self._scene_canvas.pixel_scale
+            y_pos = render_size[1] - (
+                event.pos[1] * self._scene_canvas.pixel_scale
+            )
+
+            visual_layer = self.layer_to_visual[layer]
+
```

Comment:
```suggestion
```

These lines can just be removed because you already created `visual_layer` outside the if block?